packages/mdc-toolbar/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MDCComponent} from '@material/base';
+
+import MDCToolbarFoundation from './foundation';
+import * as util from './util';
+
+export {MDCToolbarFoundation};
+export {util};
+
+export class MDCToolbar extends MDCComponent {
+  static attachTo(root) {
+    return new MDCToolbar(root);
+  }
+
+  get flexibleRowElement_() {
+    return this.root_.querySelector(MDCToolbarFoundation.strings.FLEXIBLE_ROW_SELECTOR);
+  }
+
+  get titleElement_() {
+    return this.root_.querySelector(MDCToolbarFoundation.strings.TITLE_SELECTOR);
+  }
+
+  set fixedAdjustElement(fixedAdjustElement) {
+    this.fixedAdjustElement_ = fixedAdjustElement;
+    this.foundation_.updateAdjustElementStyles();
+  }
+
+  get fixedAdjustElement() {
+    return this.fixedAdjustElement_;
+  }
+
+  getDefaultFoundation() {
+    return new MDCToolbarFoundation({
+      hasClass: (className) => this.root_.classList.contains(className),
+      addClass: (className) => this.root_.classList.add(className),
+      removeClass: (className) => this.root_.classList.remove(className),
+      registerScrollHandler: (handler) => window.addEventListener('scroll', handler, util.applyPassive()),
+      deregisterScrollHandler: (handler) => window.removeEventListener('scroll', handler, util.applyPassive()),
+      registerResizeHandler: (handler) => window.addEventListener('resize', handler),
+      deregisterResizeHandler: (handler) => window.removeEventListener('resize', handler),
+      getViewportWidth: () => window.innerWidth,
+      getViewportScrollY: () => window.pageYOffset,
+      getOffsetHeight: () => this.root_.offsetHeight,
+      getFlexibleRowElementOffsetHeight: () => this.flexibleRowElement_.offsetHeight,
+      notifyChange: (evtData) => this.emit('MDCToolbar:change', evtData),
+      setStyle: (property, value) => this.root_.style[property] = value,
```

Comment:
usually we use `this.root_.style.setProperty(property, value)` here since it allows us to use dash-case property names, and thus support custom properties.